src/tests/unit/tests/DetailsView/components/overview-content/help-links.test.tsx

```diff
@@ -4,14 +4,17 @@ import { shallow } from 'enzyme';
 import * as React from 'react';
 import { Mock } from 'typemoq';
 
-import { HelpLinks } from '../../../../../../DetailsView/components/overview-content/overview-help-links';
-import { HelpLinkDeps, OverviewHelpProps } from '../../../../../../DetailsView/components/overview-content/overview-help-section';
+import { HelpLinks } from '../../../../../../DetailsView/components/overview-content/help-links';
+import {
+    OverviewHelpSectionDeps,
+    OverviewHelpSectionProps,
+} from '../../../../../../DetailsView/components/overview-content/overview-help-section';
 
-describe('OverviewHelpLinks', () => {
-    const deps = Mock.ofType<HelpLinkDeps>().object;
+describe('HelpLinks', () => {
+    const deps = Mock.ofType<OverviewHelpSectionDeps>().object;
```

Comment:
Did you mean for this test of HelpLinks to have a dependency on OverviewHelpSectionProps/Deps?

Reply:
Nop, my mistake. Good catch!!!